package/Dockerfile

```diff
@@ -6,14 +6,14 @@ FROM alpine:3.12 AS build
 ARG ARCH=amd64
 RUN apk add -U curl xz
 ENV KUBECTL_VERSION v1.19.7
-ENV K9S_VERSION v0.24.14
-ENV KUSTOMIZE_VERSION v3.10.0
+ENV K9S_VERSION v0.25.3
+ENV KUSTOMIZE_VERSION v4.4.1
 ENV KUSTOMIZE_URL=https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_${ARCH}.tar.gz
 RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/${ARCH}/kubectl && \
     chmod +x kubectl
 RUN if [ "${ARCH}" = "amd64" ]; then ARCH=x86_64; fi && \
     curl -sfL https://github.com/derailed/k9s/releases/download/${K9S_VERSION}/k9s_Linux_${ARCH}.tar.gz | tar xvzf -
```

Comment:
Later on line 34, `k9s` is copied from the build stage for all architectures. Will the copying fail if `ARCH=s390x`, since this download happens only for `ARCH=amd64`?

Reply:
The curl command runs for all architectures, but for amd64 ARCH is changed to x86_64. if condition ends in line no 14 only.